src/test/java/easytests/api/v1/controllers/UsersControllerTest.java

```diff
@@ -137,12 +138,60 @@ public void testListForbidden() throws Exception {
     /**
      * testShowWithSubjectsSuccess
      */
-    /**
-     * testDeleteSuccess
-     */
-    /**
-     * testDeleteFailed
-     */
+    @Test
+    public void testDeleteSuccess() throws Exception {
+        final UserModelInterface userAdminModel = new UserModel();
+        userAdminModel.map(this.usersSupport.getAdminUser());
+        when(this.sessionService.getUserModel()).thenReturn(userAdminModel);
+
+        final UserModelInterface userModel = new UserModel();
+        userModel.map(this.usersSupport.getEntityFixtureMock(0));
+        when(this.usersService.find(any(), any())).thenReturn(userModel);
+
+        final ArgumentCaptor<UserModelInterface> argumentCaptor = ArgumentCaptor.forClass(UserModelInterface.class);
+
+        mvc.perform(delete("/v1/users/1")
+                .contentType(MediaType.APPLICATION_JSON)
+        )
+                .andExpect(status().isOk())
+                .andExpect(content().string(""))
+                .andReturn();
+        verify(this.usersService, times(1)).delete(argumentCaptor.capture());
+        this.usersSupport.assertEquals(argumentCaptor.getValue(), userModel);
+    }
+
+    @Test
+    public void testDeleteForbidden() throws Exception {
+        final UserModelInterface userModel = new UserModel();
+        userModel.map(this.usersSupport.getEntityFixtureMock(0));
+        when(this.usersService.find(any(), any())).thenReturn(userModel);
+
+        final UserModelInterface userNotAdminModel = new UserModel();
```

Comment:
may be it is a good idea to move mocking current user as admin and as not admin into separate functions in test.
i.e. withAdmin  withNotAdmin and after use that functions in all test methods.